tests/test-sources/plugins/by-name/openscad/default.nix

```diff
@@ -1,4 +1,8 @@
-{
+{ lib, pkgs, ... }:
+# TODO: Added 2025-05-28
+# `opencsg` fails to build on darwin
+# https://github.com/NixOS/nixpkgs/issues/411700
+lib.optionalAttrs (!pkgs.stdenv.hostPlatform.isDarwin) {
```

Comment:
Commit says "disable on `x86_64-darwin`" but code says "disable on `isDarwin`"

Reply:
Good catch. Initially, I thought that it was only on x86.

Fixed the commit message.